src_web/kamaho-shokusu/src/Policy/AuditLogPolicy.php

```
@@ -14,12 +14,12 @@
  */
 class AuditLogPolicy
 {
-    public function canIndex(?IdentityInterface $user, mixed $resource): bool
+    public function canIndex(?IdentityInterface $user, \App\Controller\AuditLogController $resource): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$resource'. (undefined)

(UnusedFormalParameter)